src/backends/perf_pt/perf_pt.c

```diff
@@ -434,31 +423,32 @@ perf_pt_start_tracer(struct tracer_conf *tr_conf)
     tr_ctx->stop_fd_rd = stop_fds[0];
     tr_ctx->stop_fd_wr = stop_fds[1];
 
-    // Use a semaphore to wait for the tracer to be ready.
-    sem_t tracer_init_sem;
-    if (sem_init(&tracer_init_sem, 0, 0) == -1) {
+    // Allocate buffer to copy the trace into (currently the same size as the AUX buffer).
```

Comment:
Not sure that "AUX" should be capitalised (it's not an acronym).

Reply:
can do.

Reply:
On second thoughts, the manual page calls it `AUX`.

E.g.

> The AUX region allows mmaping a separate sample buffer for high-bandwidth data streams (separate from the main perf sample buffer).

Reply:
Fair enough. I vaguely remember somewhere where it wasn't capitalised thus, but I might be wrong. Maybe quickly check?

Reply:
Fixed. Thanks.